docs/concepts/services-networking/service.md

```diff
@@ -429,10 +429,18 @@ with the user-specified `loadBalancerIP`. If the `loadBalancerIP` field is not s
 an ephemeral IP will be assigned to the loadBalancer. If the `loadBalancerIP` is specified, but the
 cloud provider does not support the feature, the field will be ignored.
 
-#### Internal load balancer on AWS
+Special notes for Azure: To use user-specified public type `loadBalancerIP`, a static type
+public IP address resource needs to be created at first, and it should be in the same resource
```

Comment:
public IP address resource needs to be created first, and it should be in the same resource